test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpExplicitExpressionTest/ShouldAcceptEscapedQuoteInNonVerbatimStrings.stree.txt

```diff
@@ -1,9 +1,12 @@
-Expression block - Gen<Expr> - 7 - (0:0,0)
-    Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-        SyntaxKind.Transition;[@];
-    MetaCode span - Gen<None> - [(] - SpanEditHandler;Accepts:None - (1:0,1) - Tokens:1
-        SyntaxKind.LeftParenthesis;[(];
-    Code span - Gen<Expr> - ["\""] - SpanEditHandler;Accepts:Any - (2:0,2) - Tokens:1
-        SyntaxKind.StringLiteral;["\""];
-    MetaCode span - Gen<None> - [)] - SpanEditHandler;Accepts:None - (6:0,6) - Tokens:1
-        SyntaxKind.RightParenthesis;[)];
+CSharpCodeBlock - [0..7)::7 - [@("\"")]
+    CSharpExplicitExpression - [0..7)::7
+        CSharpTransition - [0..1)::1 - Gen<None> - SpanEditHandler;Accepts:None
+            Transition;[@];
+        CSharpExplicitExpressionBody - [1..7)::6
+            RazorMetaCode - [1..2)::1 - Gen<None> - SpanEditHandler;Accepts:None
```

Comment:
Made a comment about this in a different PR but it probably makes more sense for the () to live outside of the body.

Reply:
Wait, actually you use the CodeBlock to contain all of the actual content of the expression. Nevermind, this is fine.